common/src/main/java/com/linecorp/line/auth/fido/fido2/common/mdsv3/metadata/CodeAccuracyDescriptor.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2018 LINE Corporation. All rights reserved.
+ * LINE Corporation PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
+
+package com.linecorp.line.auth.fido.fido2.common.mdsv3.metadata;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CodeAccuracyDescriptor {
+    private int base;
+    private int minLength;
+    private Integer maxRetries;
+    private Integer blockSlowdown;
```

Comment:
All those members in the spec are defined as `unsigned short` but here we have a mix of `int` and `Integer`. Is there a reason for this?

Reply:
Fixed in 17a944d.